src/lambda_bucket.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import os
+from troposphere import Template, Parameter, Ref, GetAtt, Join
+from troposphere import s3, kms
+
+
+def get_cf_template():
+    t = Template()
+    t.set_description('Creates an S3 bucket that can be used when uploading lambda packages')
+
+    bucket_name_param = t.add_parameter(Parameter(
+        'BucketName',
+        Description='Name of the S3 bucket that will contain lambda packages',
+        Type='String',
+    ))
+
+    # aws_account_ids_param = t.add_parameter(Parameter(
+    #     'AwsAccountIds',
+    #     Description='List of account ids from which the Lambda services will download the lambda packages from',
+    #     Type='CommaDelimitedList',
+    # ))
+
+    # t.add_resource(kms.Key(
+    #     'LambdaBucketKey',
+    #
+    # ))
```

Comment:
Should these commented lines be removed?

Reply:
yes